bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
```

Comment:
Could you add the shebang so it could be directly executed as
`generate_bindings.py`
instead of 
`python generate_bindings.py`
?

Reply:
Done